packages/frontend/src/main/components/viewer/CommentsOverlay.vue

```diff
@@ -484,6 +501,24 @@ export default {
         }
       }
     },
+    nextComment(comment) {
+      const index = this.activeComments.findIndex((c) => c.id === comment.id)
```

Comment:
Both of these methods are almost identical, you could merge them into a single function that handles all of the scenarios in it (idx === -1, idx === MAX, idx === 0)

Reply:
done